### ISSUE TITLE
Chromatography is a major contributor to variance, we should add metadata from that

### ISSUE THREAD

**Issue by mwalzer** [bigbio/proteomics-metadata-standard#33]
see #32

**Comment by mwalzer** [comment]
But do we need Temperature and Pressure?

**Comment by mwalzer** [comment]
Is it conceivable to drive non-linear gradients?

**Comment by ypriverol** [comment]
@mwalzer can you do a PR where you suggest in the specification how to encode additional metadata related with the Chromatograms, here the place to put the information (https://github.com/bigbio/proteomics-metadata-standard/blob/master/experimental-design/README.adoc#4-msrun-technical-details-properties)

**Comment by lisavetasol** [comment]
You have added many things about chromatography, but it seems the **fractionation type** is still missing. It could be the same types as for separation before MS (like RP, HILIC and so on), but with additional common fractionation types (high pH RP, SDS PAGE, off-gel electrophoresis, SCX). Also, maybe consider adding the information about concatenation method (with concatenation / without concatenation).

**Comment by ypriverol** [comment]
This ontology term [separation](https://www.ebi.ac.uk/ols/ontologies/pride/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPRIDE_0000022&viewMode=All&siblings=false) 

Can be used to add different separation methods. We can add a new term called `fractionation type` and under that term all the methods we would like to add. Can you list @lisavetasol all the methods you would like to add. ?

**Comment by lisavetasol** [comment]
I'm mostly familiar with the bottom-up approach and probably there are some other types for top-down/middle-down, so I suspect my list isn't complete. 
1. SDS PAGE / Gel electrophoresis (the second one is more general)
2. Off-gel electrophoresis
3. HiRIEF / Isoelectric focusing (the second one is more general)
4. HILIC (ERLIC can be a separate type or just a sub-type)
5. Ion-exchange chromatography (SCX/WCX and SAX/WAX can be sub-types)
6. Size-exclusion chromatography
7. RP (high-pH RP can be a separate type or just a sub-type)
Also, it should be possible to specify two types together for 2D separation.

**Comment by ypriverol** [comment]
@mwalzer will be updating our ontology.

**Comment by ypriverol** [comment]
This has been done. https://github.com/PRIDE-Utilities/pride-ontology/blob/master/pride_cv.obo#L3673 @lisavetasol check all the terms.

**Comment by lisavetasol** [comment]
Looks great! 
1. I think this one 
`id: PRIDE:0000556
name: Ion-exchange chromatography `
is redundant since you already specified SCX, SCA, WSX and WSA. Propose to delete it or add a relation `is_a:  PRIDE:0000556!Ion-exchange chromatography`. 
2. I wouldn't use LC for `id: PRIDE:0000562
name: Normal-phase liquid chromatography (LC)` because LC usually means just liquid chromatography (not a normal-phase one). So a better way would be "NP" to be consistent with "RP". 
3. `id: PRIDE:0000554
name: Hydrophilic interaction chromatography (HILIC)` can be as well  `is_a: PRIDE:0000551 ! Separation method`. 
4. `[Term]
id: PRIDE:0000122
name: Two Dimensional Polyacrylamide Gel Electrophoresis` should be `is_a: PRIDE:0000550 ! Fractionation method` but not separation according to @mwalzer  
>  I assumed separation is where can be directly coupled to MS.
5. Not sure if `id: PRIDE:0000565
name: High-performance liquid chromatography` should have is_a relations, but they are missing. If it's meant to be as Separation and Fractionation, I think it may be confusing since there are already specific types (RP, hpHRP, HILIC and so on).  

Beside these, it looks good to me :)

**Comment by ypriverol** [comment]
@lisavetasol we have solved all the issues in the last PR.

**Comment by lisavetasol** [comment]
I think these ones 
`id: PRIDE:0000562
name: Normal-phase liquid chromatography (NP)`
`id: PRIDE:0000563
name: Reversed-phase chromatography (RP)`
aren't ion-exchange ones. They are 
`is_a: PRIDE:0000565 name: High-performance liquid chromatography` if you wish or just  
`is_a: PRIDE:0000550 ! Fractionation method` and `is_a: PRIDE:0000551 ! Separation method`

**Comment by ypriverol** [comment]
Done thanks for the feedback @lisavetasol

**Comment by ypriverol** [comment]
@mwalzer @lisavetasol the new terms have been updated in PRIDE https://www.ebi.ac.uk/ols/ontologies/pride . Please can you have a look?

**Comment by lisavetasol** [comment]
Cool, I think it's enough terms :) 
Just maybe the relations, between them, are a little bit incorrect. I sketched the way I think they are related. Not sure if that's a better way, but I highlighted in red the stuff I think may be confusing in the current version. 
![Pride ontology fractionation methods](https://user-images.githubusercontent.com/41292192/80700106-6e3b3180-8add-11ea-90ed-b73e6ddc4385.png)
